### ISSUE TITLE
Using aria-owns with late-mounted descendants

### ISSUE THREAD

**Issue by mpaiva** [w3c/aria-practices#2771]
I'm creating this issue propose an example to explain how to use the `aria-busy` to track descendant elements status with `aria-activedescendant` that relate to ancestors elements through `aria-owns`.

This detailed example can help web developers keeping track of the loading state of a descendant element, which came to our attention in the ARIA-WG with issues [1970](https://github.com/w3c/aria/issues/1970) and [1956](https://github.com/w3c/aria/issues/1956) asking us to provide additional language clarification. 

I am currently drafting the new language and a code example with appropriate documentation: 

https://dev.to/mpaiva/using-aria-owns-with-late-mounted-descendants-nma

Could you please provide some guidance on how to best do this?

**Comment by brennanyoung** [comment]
The most correct way to do it is to clone this repository, make changes to your own version, and then make a pull request from yours to this one.

**Comment by jongund** [comment]
Thank you for your interest in providing examples on the use of `aria-busy`.

Some issue that need to be clarified in your example:
1. The listboxes in your example do not have any keyboard support, see the listbox examples for adding keyboard support. [see Select Only Listbox example]https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only/]
2. `aria-busy` should be on the list box container (e.g. role=listbox element), not on the role=option elements. 
3. It seems right now the fights are simple lists, what is the advantage of making them a listbox in this example?
